teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly refer to my comment above for the necessary change that should be made in the `Teacher` class 👍

Reply:
👍